pkg/api/graph/types.go

```diff
@@ -222,8 +223,14 @@ func ImageStreamTag(g MutableUniqueGraph, namespace, name, tag string) graph.Nod
 	if len(tag) == 0 {
 		tag = "latest"
 	}
+	var uname UniqueName
+	if strings.Index(name, ":") == -1 && strings.Index(name, "@") == -1 {
```

Comment:
you'd think, right?  but how we generate the unique name depends on what's already in the name.